### PR TITLE
Update projects.rst

### PR DESCRIPTION
**Pull request by rezafuru**
Extend list of projects with FrankenSplit

### CODE REVIEW

File: docs/source/projects.rst [hunk 1]
```diff
@@ -27,6 +27,16 @@ It is pip-installable and published as a PyPI package i.e., you can install it b
 Papers
 *****
 
+FrankenSplit: Efficient Neural Feature Compression With Shallow Variational Bottleneck Injection for Mobile Edge Computing
+----
+* Author(s): Alireza Furutanpey, Philipp Raith, Schahram Dustdar
+* Venue: IEEE Transactions on Mobile Computing
+* PDF: `Paper <https://ieeexplore.ieee.org/document/10480247/>`_
+* Code: `GitHub <https://github.com/rezafuru/FrankenSplitl>`_
```
Comment (yoshitomo-matsubara) [inline docs/source/projects.rst:35]: The link is broken. It should be `https://github.com/rezafuru/FrankenSplit`

**Review by yoshitomo-matsubara — CHANGES_REQUESTED** [review]
Hi @rezafuru 

Thanks for the PR. It looks good. Please fix the broken GitHub link as requested above.

File: docs/source/projects.rst (hunk 1 above)
Comment (rezafuru) [reply docs/source/projects.rst]: Apologies, should be fine now

**Comment by yoshitomo-matsubara** [comment]
It looks good to me. I will merge this PR soon

**Review by yoshitomo-matsubara — APPROVED** [review]
It looks good to me. I will merge this PR soon

**Pull request merged**